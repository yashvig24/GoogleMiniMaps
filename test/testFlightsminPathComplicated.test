# Tests find minimum-cost path on flights graph.

# Load sports.tsv
LoadGraph flights flights.tsv

# Test find minimum-cost path not shortest path
FindPath flights New-York Las-Vegas