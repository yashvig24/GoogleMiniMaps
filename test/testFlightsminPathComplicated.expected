# Tests find minimum-cost path on flights graph.

# Load sports.tsv
loaded graph flights

# Test find minimum-cost path not shortest path
path from New-York to Las-Vegas:
New-York to San-Fran with weight 0.500
San-Fran to Las-Vegas with weight 1.000
total cost: 1.500
